### PR TITLE
New Student entry.

### PR DESCRIPTION
**Pull request by ameybh**
Thank you for your pull request! Please read and understand everything below

Please acknowledge that you have read and understood each of the following before submitting your pull request.

# _Do not_ delete any text other than where you are instructed.

**Maintainers: If any of these are left unchecked, do not merge.**

**Students: If one of them is not applicable to you -- Please check it anyways. DO NOT REMOVE ANYTHING!**

**You may check them after opening your pull request.**

You may do so by changing each `[ ]` to `[x]` Please take note of the whitespace as it matters.

- [ ] Read and understood (see CONTRIBUTING.md)
- [ ] Included a live link or preview screenshot
- [ ] Images are `240 x 240` [w x h]
- [ ] Resolved merge conflicts
- [ ] Included a description of my change below

# Things done in this Pull Request

- Replace this with your change description

### CODE REVIEW

File: _data/student_projects.yml [hunk 1]
```diff
@@ -10,7 +10,12 @@
   github: https://github.com/0x48piraj/FOSSASIATask_FlaskAppForm
   author: Piyush Raj | 0x48piraj
   image: flask_0x48piraj.png
-
+  
+- name: Amey Bhavsar
+  github: https://github.com/ameybhavsar24/gci17.fossasia.org
+  author: Amey Bhavsar | ameybhavsar24
+  image: amey.png
```
Comment (gabru-md) [inline _data/student_projects.yml:17]: Image not found.

File: _data/students.yml [hunk 2]
```diff
@@ -45,6 +45,11 @@
   image: abishek.jpg
   lat: 9.9312328
   lng: 76.26730409999999
+  
+  -name: Amey Bhavsar
+  image: amey.jpg
```
Comment (gabru-md) [inline _data/students.yml:50]: Image not found

**Review by gabru-md — CHANGES_REQUESTED** [review]
Please add the images that you have mentioned in the PR.

**Review by 0x48piraj — CHANGES_REQUESTED** [review]
Remove the conflicts!

**Review by anshumanv — CHANGES_REQUESTED** [review]
Please upload appropriate images in respective directories.

**Review by the-codingninja — CHANGES_REQUESTED** [review]
Resolve conflicts and add appropriate image

**Comment by STEdirisuriya** [comment]
- Add a description
- Check requirements in PR introduction
- Add a image
- Resolve conflicts
- If you can squash after doing those things.

**Review by supunTE — CHANGES_REQUESTED** [review]
Add a description
    Check requirements in PR introduction
    Add a image
    Resolve conflicts
    If you can squash after doing those things.

**Review by ParthS007 — CHANGES_REQUESTED** [review]
@ameybhavsar24  Please follow `README.md` on how to add blog an student and update your PR 👍

**Pull request closed without merging**